__tests__/formatter-test.ts

```diff
@@ -0,0 +1,47 @@
+import { ESLint } from 'eslint';
+import fs from 'fs';
+import path from 'path';
+import tmp from 'tmp';
+import { formatResultsAsync } from '../src/formatter';
+
+describe('formatter', () => {
+  const INITIAL_ENV = process.env;
+  let tmpDir: tmp.DirResult;
+
+  beforeEach(() => {
+    jest.resetModules();
+    tmpDir = tmp.dirSync({ unsafeCleanup: true });
+    process.env = { ...INITIAL_ENV, ESLINT_TODO_DIR: tmpDir.name };
+  });
+
+  afterEach(() => {
+    tmpDir.removeCallback();
+  });
+
+  afterAll(() => {
+    process.env = INITIAL_ENV;
+  });
+
+  it('SHOULD NOT generate a TODO dir with todo files when UPDATE_TODO is set to 0', async () => {
+    // set the variables
+    process.env.UPDATE_TODO = '0';
+    const results = require('./__fixtures__/eslint-with-errors.json') as ESLint.LintResult[];
```

Comment:
So the problem is that the code mutates this object tree which affects other tests. The reason why I'm requiring and also resetting module the module on `beforeEach` is because I want a brand new copy for every test.

Reply:
Ok. I fixed this, I figured using `resetModules` with `require` in tests is brittle and prone to problems, so I decided to just read the JSON directly instead, this way we won't have to do any of those weird calls.